Makefile.devel

```diff
@@ -31,6 +31,11 @@ man: third-party-chpldoc-venv FORCE
 man-chpldoc: FORCE
 	cd man && $(MAKE) chpldoc
 
+man-chapel: FORCE
+	cd doc/sphinx && $(MAKE) man-chapel
+	mkdir -p man/man3
+	cp doc/sphinx/build/man/chapel.3 man/man3/chapel.3
+
```

Comment:
I thought we already built a man page, but I'm not seeing the rule for that here.  Do you know where it lives?  And is there a better way to distinguish what this is doing in the name?